### PR TITLE
ref(replays): Absolute imports in all the replays files

### PR DESCRIPTION
**Pull request by ryan953**
**Motivation:**
I was renaming some stuff:
```
- import {Replay} from './types';
+ import {DiscoReplayListItem} from './types';
```
but it was annoying because, before i jumped into the change, I wanted to find all the places that did the import and only do the rename if it’s a small list.
So how do I find all the places that use `Replay` or `types`, there are a lot of search hits for a naive search. Another approach might've been to rename the thing and see what breaks, but that's noisy too, and the broken lines will not always be the lines that need to be updated.

Basically, i'm asserting that 'relative imports' make this tough.
Sometimes we’ve got `import foo from './foo';` and other times it’s crawling up the tree `import foo from '../foo';` even.

So, at least in this corner of the world, I think some standardization is good. VSCode will auto-update imports when a file is renamed... but sometimes I just want to grep around and figure out the size of a problem before messing up my working copy.

### CODE REVIEW

**Review by billyvg — APPROVED** [review]
We should gauge interest in https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-relative-packages.md -- it's `--fixable`

**Comment by ryan953** [comment]
> We should gauge interest in https://github.com/import-js/eslint-plugin-import/blob/HEAD/docs/rules/no-relative-packages.md -- it's `--fixable`

Yeah. I've started a FE TSC proposal: https://www.notion.so/sentry/Inconsistent-relative-absolute-import-paths-078ca61c73ff4908a49ab40db2ea8dd2
